cmd/manager/main.go

```diff
@@ -75,6 +82,9 @@ var (
 			}
 			if flags.controllerOptions.Namespace == "" {
 				flags.controllerOptions.Namespace = os.Getenv("POD_NAMESPACE")
+				if flags.controllerOptions.Namespace == "" {
```

Comment:
namespace "" has a special meaning - it means all namespaces. why are we removing the option to listen on all namespaces?